cityofaustin/atd-vz-data
# Crash details page: charges related records table

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17319

I also updated the queries this page uses so that all the components on the crash details page can pull data from one top level crash query. The query refetches whenever any edits are made so the header widgets will update if for example an injury severity changes from non fatal to fatal.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Follow [the steps](https://github.com/cityofaustin/atd-vz-data/pull/1440) to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
2. Apply the hasura migrations and metadata from this branch
3. Go to a crash with charges, see that the charges table is now pulling data from the new data model.
4. Test updating a persons injury severity and see that the header widgets are updating automatically as well.

---
#### Ship list
